### PR TITLE
Analysis sketch

### PR DESCRIPTION
**Pull request by matthew-brett**
See the file `on_the_project.md` for more instructions on how to use this PR.
    
For your repo, I see you have already filled in the outfile.py file, so I didn't merge those changes.  So disregard the comments about the DVARS outlier detection in outfile.py, in on_the_project.py.

### CODE REVIEW

File: on_the_project.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+You can usually stick to the standard stuff, but you may need to consult the
+Github documents if you want to do something slightly more fancy, like a table.
+
+## Making sure you're ready
+
+To be ready to get going on your project you need to make sure you have merged these three PRs:
+
+* "add-dvars"
+* "Add machinery to install module directory."
+* "Fix use of Path in find_outliers script"
```
Comment (sdiebolt) [inline on_the_project.md:78]: @matthew-brett We didn't get this PR, but I guess it's about the bug we found in c240d0b1941ca7b782dbacb6c2cb7a193244fbe4?

Comment (FelipeCybis) [reply on_the_project.md:78]: I guess so. Also for "add-dvars" it was our PR #7.

**Review by FelipeCybis — APPROVED** [review]
All good for me. I am ok on merging this PR and start sketching our analysis plan via issue #10. 
@sdiebolt @nipraxis-fall-2022/instructors

**Comment by sdiebolt** [comment]
LGTM!

**Pull request merged**